src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -1556,6 +1556,13 @@ public static List<SitePinInst> unrouteCellPinSiteRouting(Cell cell, String logi
             Net net = siteInst.getNetFromSiteWire(belPin.getSiteWireName());
             if (net == null)
                 return Collections.emptyList();
+            else {
+                // Check if net is connected to logical cell
+                EDIFPortInst portInst = cell.getEDIFCellInst().getPortInst(logicalPinName);
```

Comment:
How should `cell.getEDIFCellInst() == null` be handled, as for encrypted designs? (outside of not NPE-ing)

Reply:
Thanks, I added a check for that.